src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -4097,6 +4108,7 @@ void CalcISO15099WindowIntConvCoeff(EnergyPlusData &state,
     // Establish some lower limit to avoid a zero convection coefficient (and potential divide by zero problems)
     if (state.dataHeatBalSurf->SurfHConvInt(SurfNum) < state.dataHeatBal->LowHConvLimit)
         state.dataHeatBalSurf->SurfHConvInt(SurfNum) = state.dataHeatBal->LowHConvLimit;
+    state.dataHeatBalSurf->SurfHConvInt(SurfNum) *= state.dataHeatBalSurf->SurfWinCoeffAdjRatioIn(SurfNum);
```

Comment:
Note that this appears to violate ISO 15099.  Although I see why it is desired/necessary to get the desired outcome here, doing this in such a quiet way is bit difficult to justify.

Reply:
@EnergyArchmage I understand the concern. But based on the design of this feature, the adjusted coefficients does not represent the original glazings derived by the simple glazing system, they should represent the overall coefficients of the glazing and imaginary highly conductive frames, of which the geometry and properties cannot be easily solved by the simple glazing U alone.